pkg/grouper/package_grouper.go

```diff
@@ -53,37 +56,42 @@ func GroupByPURL(packageSources []models.PackageSource) map[string]models.Packag
 }
 
 func isLocationExtractedSuccessfully(filePosition models.FilePosition) bool {
-	return filePosition.Line.Start > 0 && filePosition.Line.End > 0 && filePosition.Column.Start > 0 && filePosition.Column.End > 0
+	return filePosition.Line.Start > 0 && filePosition.Line.End > 0 && filePosition.Column.Start > 0 && filePosition.Column.End > 0 && filePosition.Filename != ""
 }
 
-func extractPackageLocations(pkgSource models.SourceInfo, pkgInfos models.PackageInfo) models.PackageLocations {
+func removeHostPath(source models.SourceInfo, absolutePath string) string {
+	path := filepath.ToSlash(strings.TrimPrefix(absolutePath, source.HostPath))
+	if !strings.HasPrefix(source.Path, "/") {
+		path = strings.TrimPrefix(path, "/")
+	}
```

Comment:
I think this condition is inverted right ? 
It is doing if the path is not prefixed by "/" remove it

Reply:
It is related with the logic explained in the previous comment:
- If source.Path does NOT start with "/", it means the PathRelativeToScanDir flag is set to true
- This means the filename reported will also require to NOT start with "/", so that's why its trimmed

```go
// osvscanner.go - removeHostPath
if shouldBeRelative {
	pkg.Source.Path = strings.TrimPrefix(pkg.Source.Path, "/")
}
```